packages/compass-query-bar/src/stores/query-bar-reducer.spec.ts

```diff
@@ -226,6 +226,21 @@ describe('queryBarReducer', function () {
         '{ _id: -1 }'
       );
     });
+
+    it('should auto expand when the query contains extra options', function () {
```

Comment:
do we need a counter test to make sure the qb doesn't expand if options aren't present?

Reply:
There's a counter test inside of this test already, it would have been better practice to explicitly have it as a separate test though, good callout.